src/map/GMap.js

```diff
@@ -455,6 +649,66 @@
     GMap.prototype.zoomToFit = function () {
         return this.zoomTo(this.data());
     };
+    
+    GMap.prototype.drawingOptions = function(_) {
+        if(typeof(_) === "object") {
```

Comment:
I don't think I would bother with this test (since we don't test params like this anywhere else).